config.py

```diff
@@ -91,6 +92,9 @@ def load_environment_variables(self):
         self.FINALIZATION_TIME_BORDER: int = int(
             os.getenv("ZSEQUENCER_FINALIZATION_TIME_BORDER", "120")
         )
+        self.AGGREGATION_TIMEOUT: int = int(
+            os.getenv("ZSEQUENCER_SIGNATURES_AGGREGATION_TIMEOUT", "120")
```

Comment:
Let's reduce this default value to 5 seconds

Reply:
Fixed.